docs/api/response.md

```diff
@@ -130,9 +130,18 @@ ctx.response.status = 204;
   - `Buffer` written
   - `Stream` piped
   - `Object` || `Array` json-stringified
-  - `null` no content response
+  - `null` || `undfined` no content response
+
+If `response.status` has not been set, Koa will automatically set the status to `200` or `204` depending on `response.body`. Specifically, if `response.body` has not been set or has been set as `null` || `undfined`, which means no content response, Koa will automatically set `response.status` to `204`. If you really want to send no content response with other status, you should override the `204` status as the following way:
```

Comment:
`undfined` misspelled.

Reply:
I would remove `which means no content response`, the reason to why null or undefined it set is ambiguous to Koa - but it interprets it as 204 No Content as you wrote.

Reply:
NIT: I would not use `||`'s for documentation text, rather use explicit `or`

Reply:
Thanks. That was updated. I also thought about `or` against `||` as definition listed above. Now i change to use `or` for documentation text.